resources/js/ziggy.d.ts

```diff
@@ -2,14 +2,23 @@
 // See the LICENCE file in the repository root for full licence text.
 
 declare module 'ziggy' {
-  interface ZiggyClass {
+  // Reduced version of unexported ziggy Config for typing when setting the port and url.
+  // Using Parameters<typeof route>[3] doesn't quite work since route has overloaded parameters.
+  interface ZiggyGlobal {
```

Comment:
this?

```typescript
  import route, { RouteList } from 'ziggy-js';

  export const Ziggy: NonNullable<Parameters<typeof route<keyof RouteList>>[3]>;
```

Reply:
Turns out I never tried `<keyof RouteList>` here since I only started using it after trying to strip the parameter for the wrapper 🤔